### DISCUSSION TITLE
On form validation error, scroll to hidden element or ErrorMessage of the element

### DISCUSSION THREAD

**Discussion by Kukiezi** [react-hook-form/react-hook-form#9408] [Q&A]
**Version Number**
7.38.0

**Question**
I have a form in which some elements are using hidden input, and I am setting values to them with `setValues` method. In the case of validation error, the from will get confused and not scroll to the input. My understanding is that you can't focus on the hidden input, and that is why it doesn't scroll there. Is there a workaround someone could suggest for this?

**Comment by bluebill1049** [comment]
It would be tricky, but not impossible you may need to store that ref for the parent element instead of the input itself, and manually scroll into those element eg `div, section` and etc
